cabal.project

```diff
@@ -57,11 +57,14 @@ packages:
   eras/byron/crypto
   libs/cardano-ledger-conformance
   libs/cardano-ledger-test
-  libs/plutus-preprocessor
   libs/ledger-state
   libs/constrained-generators
   libs/cardano-ledger-repl-environment
 
+if ((impl(ghc >=9.6) && impl(ghc <9.7)) && !os(windows))
+  packages:
+    libs/plutus-preprocessor
+
```

Comment:
Revert back to how it was before. This was not part of the task and I do not see any reason why would we do that. It is up to individual package to decide if it is buildable or not with a specific GHC version
```suggestion
```